### PR TITLE
feat(integration): Anthropic integration

### PR DESCRIPTION
**Pull request by tcapelle**
- Add basic weave.op support for anthropic SDK
- Function calling is working
- Basic docs copied from MistralAI ones.

We are missing:
- The Bedrock and VertexAI integrations will come in separate PRs

### CODE REVIEW

**Comment by tcapelle** [comment]
I have read the CLA Document and I hereby sign the CLA

File: weave/integrations/anthropic/anthropic_sdk.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            acc.stop_reason = value.delta.stop_reason
+        if hasattr(value.delta, "stop_sequence") and value.delta.stop_sequence:
+            acc.stop_sequence = value.delta.stop_sequence
+        if hasattr(value, "usage") and value.usage.output_tokens:
+            acc.usage.output_tokens = value.usage.output_tokens
+
+    return acc
+
+
+# TODO: Add accumulator for beta.messages for tool usage
```
Comment (tssweeney) [inline weave/integrations/anthropic/anthropic_sdk.py:64]: is this still required? if not, please elaborate for future devs

File: docs/docs/guides/ecosystem/imgs/anthropic_ops.png
Comment (tssweeney) [inline docs/docs/guides/ecosystem/imgs/anthropic_ops.png:1]: Nit for call images - would be nice to show the W&B header for brand purposes

Comment (tcapelle) [reply docs/docs/guides/ecosystem/imgs/anthropic_ops.png:1]: It looks worst as it is not white, also I am being consistent with the OpenAI one.

File: weave/integrations/anthropic/anthropic_sdk.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    [
+        # Patch the sync messages.create method for all messages.create methods
+        SymbolPatcher(
+            lambda: importlib.import_module("anthropic.resources.messages"),
+            "Messages.create",
+            create_wrapper(name="Messages.create"),
+        ),
+        SymbolPatcher(
+            lambda: importlib.import_module("anthropic.resources.messages"),
+            "AsyncMessages.create",
```
[the remark this answers is not in the document]
Comment (tcapelle) [reply weave/integrations/anthropic/anthropic_sdk.py:96]: Ohh this is going to be long...

File: weave/integrations/anthropic/anthropic_sdk.py (hunk 1 above)
Comment (tcapelle) [reply weave/integrations/anthropic/anthropic_sdk.py:64]: no, they removed that from the SDK

**Comment by scottire** [comment]
@tcapelle can you add a section where you use anthropic within a `Model` as we do in the openAI example please.

**Review by tssweeney — APPROVED** [review]
YAY!

**Pull request merged**